src/integration-test/tests/citizen/directionsQuestionnaire/steps/directionsQuestionnaireSteps.ts

```diff
@@ -0,0 +1,44 @@
+import { SupportRequiredPage } from 'integration-test/tests/citizen/directionsQuestionnaire/pages/support-required'
+import { HearingDatesPage } from 'integration-test/tests/citizen/directionsQuestionnaire/pages/hearing-dates'
+import { HearingLocationPage } from 'integration-test/tests/citizen/directionsQuestionnaire/pages/hearing-location'
+import { UsingExpertPage } from 'integration-test/tests/citizen/directionsQuestionnaire/pages/using-expert'
+import { ExpertReportsPage } from 'integration-test/tests/citizen/directionsQuestionnaire/pages/expert-reports'
+import { SelfWitnessPage } from 'integration-test/tests/citizen/directionsQuestionnaire/pages/self-witness'
+import { OtherWitnessPage } from 'integration-test/tests/citizen/directionsQuestionnaire/pages/other-wtiness'
+import { HearingExceptionalCircumstancesPage } from 'integration-test/tests/citizen/directionsQuestionnaire/pages/hearing-exceptional-circumstances'
+
+const supportRequiredPage: SupportRequiredPage = new SupportRequiredPage()
+const hearingLocationPage: HearingLocationPage = new HearingLocationPage()
+const hearingExceptionalCircumstancesPage: HearingExceptionalCircumstancesPage = new HearingExceptionalCircumstancesPage()
+const usingExpertPage: UsingExpertPage = new UsingExpertPage()
+const expertReportsPage: ExpertReportsPage = new ExpertReportsPage()
+const selfWitnessPage: SelfWitnessPage = new SelfWitnessPage()
+const otherWitnessPage: OtherWitnessPage = new OtherWitnessPage()
+const hearingDatesPage: HearingDatesPage = new HearingDatesPage()
+
+export class DirectionsQuestionnaireSteps {
+  acceptDirectionsQuestionnaireYesJourney (): void {
```

Comment:
can you please rename this? what does the yes mean? Yes to expert reports?

Reply:
makes sense to me... 1 journey selects Yes for all DQ journey and other selects No for all DQ journey